myst_parser/mocking.py

```diff
@@ -235,12 +235,15 @@ def __getattr__(self, name):
         """This method is only be called if the attribute requested has not
         been defined. Defined attributes will not be overridden.
         """
+        cls = type(self).__name__
         if hasattr(Body, name):
-            msg = "{cls} has not yet implemented attribute '{name}'".format(
-                cls=type(self).__name__, name=name
+            msg = (
+                f"{cls} has not yet implemented attribute '{name}'. "
+                "You can parse RST directly via the `{eval-rst}` directive: "
+                "https://myst-parser.readthedocs.io/en/latest/using/syntax.html#how-directives-parse-content"  # noqa: E501
             )
-            raise MockingError(msg).with_traceback(sys.exc_info()[2])
-        msg = "{cls} has no attribute {name}".format(cls=type(self).__name__, name=name)
+        else:
```

Comment:
Can you add a comment that describes under what condition the "if" statement will return False and thus the simpler error message?

Reply:
Something like this:
```py
if hasattr(Body, name):
    ...
else:
    # The requested `name` is not a docutils Body element
    # (such as "admoniton", "block_quote", "paragraph", …)
    ...
```
?

Reply:
that seems good 👍

Reply:
I made the code a bit DRY-er but did not introduce the if-else logic, that was already there (i.e. this is a pure refactor -- no different behavior).
Maybe the original author (Chris Sewell) can explain the reasoning behind the two cases.

Reply:
Anyway I added the above comment in a commit, so should be good to go :)